src/derivers/bip39.ts

```diff
@@ -28,22 +103,26 @@ export function bip39MnemonicToMultipath(mnemonic: string): BIP39StringNode {
  * @param options - The options for creating the node.
  * @param options.path - The multi path.
  * @param options.curve - The curve to use for derivation.
+ * @param cryptographicFunctions - The cryptographic functions to use. If
+ * provided, these will be used instead of the built-in implementations.
  * @returns The node.
  */
-export async function deriveChildKey({
-  path,
-  curve,
-}: DeriveChildKeyArgs): Promise<SLIP10Node> {
+export async function deriveChildKey(
+  { path, curve }: DeriveChildKeyArgs,
+  cryptographicFunctions?: CryptographicFunctions,
+): Promise<SLIP10Node> {
   switch (curve.masterNodeGenerationSpec) {
     case 'slip10':
       return createBip39KeyFromSeed(
-        await mnemonicToSeed(path, englishWordlist),
+        await mnemonicToSeed(path, '', cryptographicFunctions),
         curve,
+        cryptographicFunctions,
       );
     case 'cip3':
       return entropyToCip3MasterNode(
         mnemonicToEntropy(path, englishWordlist),
```

Comment:
Out of scope, but seems like we are really close to being able to drop `scure-bip39` as a dep if we implement this function